packages/cli/src/models/network/TransactionController.ts

```diff
@@ -39,6 +52,21 @@ export default class TransactionController {
     }
   }
 
+  public async sendTransaction(proxyAddress: string, methodName: string, methodArgs: string[]): Promise<any> {
+    if (!this.networkFile.hasProxies({ address: proxyAddress })) {
+      throw Error(`Proxy at address ${proxyAddress} not found.`);
+    }
+
+    const { package: packageName, contract: contractName } = this.networkFile.getProxy(proxyAddress);
+    const localController = new LocalController(this.packageFile);
+    const contract = localController.getContractClass(packageName, contractName).at(proxyAddress);
+    const { method } = buildCallData(contract, methodName, methodArgs);
+
+    log.info(`Calling ${methodName} with: ${callDescription(method, methodArgs)}`);
+    const { transactionHash } = await Transactions.sendTransaction(contract.methods[methodName], methodArgs, this.txParams);
+    log.info(`Function successfully called! Transaction hash: ${transactionHash}`);
```

Comment:
We should properly handle the case where the tx reverts, showing something nice to the user, and include the transaction hash as well.

Reply:
Also, how difficult would it be to list the events emitted...?

Reply:
Should be super easy. How should we list them? do you have something in mind?